duckduckgo/iOS
# Turn SwiftLint back on

Task/Issue URL: https://app.asana.com/0/414709148257752/1196995126357495
CC:

**Description**:

This PR turns SwiftLint back on and fixes existing violations.

There are a couple things to do before this PR is ready to be merged:

* [x] Test that CI is failing PRs appropriately – will introduce a failing change on this branch and see what happens
* [x] Verify that the fixes here make sense. For example, I have a couple `type_body_length` comments in test files, but it might make more sense to disable those for the `UnitTests` directly wholesale (assuming SwiftLint can configure individual rules to exclude specific directories?)

**Steps to test this PR**:
1. Build the main app target and verify that no warnings appear
1. Introduce a violation deliberately and verify that a warning appears
